lab-lee/test/txt-read-test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const txtRead = require('../lib/txtRead.js');
+const assert = require('assert');
+
+describe('#txtRead()', function() {
+  it('should return hex value for 1.txt', function(done){
+    txtRead(function(value){
+      let result = value[0];
+      assert.ok(result === '54657874206f6e65', 'was not the right txts');
```

Comment:
...Although now we'll be using Chai, so you don't need to worry much about this!